src/lib/builtins/prompt.rs

```diff
@@ -43,16 +48,25 @@ pub(crate) fn prompt(args: &[String], shell: &mut Shell) -> Result<(), String> {
             return Err(String::from("Nothing inputted. Going back to Ion shell"));
         } else {
             let mut new_prompt = String::new();
-            let mut input_iter = input.trim().chars().peekable();
+            let mut input_iter = input.trim().chars();
             while let Some(character) = input_iter.next() {
                 match character {
                     '$' => {
                         match input_iter.next() {
                             Some('a') => { stdout.ask_color_for("${USER}", &mut new_prompt); }
                             Some('b') => { stdout.ask_color_for("${HOST}", &mut new_prompt); }
-                            Some('c') => { stdout.ask_color_for("${PWD}", &mut new_prompt); }
-                            Some('d') => { stdout.ask_color_for("${SWD}", &mut new_prompt); }
-                            _ => (),
+                            Some('c') => { 
+                                match input_iter.next() {
+                                    Some('1') => stdout.ask_color_for("${PWD}", &mut new_prompt),
+                                    Some('2') => stdout.ask_color_for("${SWD}", &mut new_prompt),
+                                    _ => continue,
+                                }
+                            }
+                            Some('d') => { 
+                                unimplemented!();
+                                stdout.ask_color_for(r#"$(top -bn1 | grep "Cpu(s)" | sed "s/.*, *\([0-9.]*\)%* id.*/\1/" | awk '{print 100 - $1"%"}')"#, &mut new_prompt);
```

Comment:
I need help figuring out to do an Ionic way of doing `$(top -bn1 | grep "Cpu(s)" | sed "s/.*, *\([0-9.]*\)%* id.*/\1/" | awk '{print 100 - $1"%"}')`.